src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -32,6 +32,40 @@ internal static void VerifyDiagnostics(string source, params DiagnosticDescripti
     [CompilerTrait(CompilerFeature.LocalFunctions)]
     public class LocalFunctionTests : LocalFunctionsTestBase
     {
+        [Fact]
+        public void TestAdHocDelegateWithParams_NotAllowRefLikeTest()
```

Comment:
It feels like this should be in a different file than `LocalFunctionTests`, for example `RefStructInterfacesTests` seems better.

Reply:
I didn't know where to define it - that would be a better place or something like "allowRefStructTests.cs"